src/components/Sidebar.tsx

```diff
@@ -53,28 +35,54 @@ const useStyles = makeStyles(() => ({
 
 const Sidebar = () => {
   const classes = useStyles();
+  const [selected, setSelected] = React.useState('1');
+
   return (
-    <Box className={`${classes.root} left`}>
+    <Box className="sidebar">
       <List disablePadding component="nav">
-        <ListItemButton selected>
+        <ListItemButton
+          selected={selected === '1'}
+          onClick={() => setSelected('1')}
+        >
           <ListItemIcon>
```

Comment:
@vidhya-metacell please refactor into DRY, e.g. use an array with the listitems and then loop, factor out the listitem jsx into a seperate file

Reply:
@zsinnema Yes i agree, i will change this